velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -929,6 +930,67 @@ PlanBuilder& PlanBuilder::localMerge(const std::vector<std::string>& keys) {
   return *this;
 }
 
+PlanBuilder& PlanBuilder::expand(
+    const std::vector<std::vector<std::string>>& projections) {
+  VELOX_CHECK(!projections.empty(), "projections must not be empty.");
+  const auto numColumns = projections[0].size();
+  const auto numRows = projections.size();
+  std::vector<std::string> aliasNames;
+  aliasNames.reserve(numColumns);
+  std::vector<std::string> originalNames;
```

Comment:
I took a closer look at this logic and I see that originalNames, types and inputRowType variables are not needed.

See "review comments" commit in https://github.com/facebookincubator/velox/compare/main...mbasmanova:velox-1:add-expand-op?expand=1

Reply:
@mbasmanova Great. Thank you so much for your code, it's very concise.